client/package.json

```diff
@@ -8,7 +8,7 @@
     "build": "esbuild ./target/scala-2.13/client-opt/main.js --bundle --outdir=../server/src/main/assets/ --minify"
   },
   "dependencies": {
-    "@drt/drt-react": "UKHomeOffice/drt-react#1.7.4",
+    "@drt/drt-react": "UKHomeOffice/drt-react#4fff455ed37b657f249e37a7c8a054ed00b2ed07",
```

Comment:
Update to a release version

Reply:
Further to this - we need to create a release in github for each drt-react version so that the version specified here corresponds to a published release in the drt-react repo